src/styles/blocks/button.css

```diff
@@ -0,0 +1,37 @@
+.information__link {
+  margin-top: var(--margin-basic);
+  
+  display: flex;
+  text-decoration: none;
+
+  justify-content: center;
+  align-items: center;
+  flex-grow: 1;
+
+  height: 38px;
+
+  background: #00acdc;
```

Comment:
Need variable here

Reply:
I remade it absolutely. In my new version block for button is absent.